stdlib/dynarray.mli

```diff
@@ -181,6 +181,23 @@ val append_iter :
     [1], [2], and then [3] at the end of [a].
     [append_iter a Queue.iter q] adds elements from the queue [q]. *)
 
+val blit : 'a t -> int -> 'a t -> int -> int -> unit
```

Comment:
I would prefer a labelled version of this function, but I won’t fight it if you tell me that it belongs to a separate `StdLabels.Dynarray` PR.

Reply:
I also prefer labels, and initially I wanted to have them, and then I realized that this introduces an inconsistency with `Array.blit` that I found hard to justify. I am happy to include labels by default if there is consensus in favor. I don't mind using labelling for the internal functions, I will update the PR with that.